R/prep_data_executive_summary.R

```diff
@@ -237,14 +239,29 @@ prep_data_executive_summary <- function(investor_name,
       entity_type = "benchmark"
     )
 
+  emissions_portfolio <- emissions_portfolio %>%
+    dplyr::mutate(entity = "portfolio")
```

Comment:
you actually want the `entity` column to be filled with "portfolio", right? as opposed to filling it with the values in a `portfolio` column?

Reply:
correct